src/entity/Lesson.ts

```diff
@@ -0,0 +1,34 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+import { Section } from './Section';
+
+@Entity('lessons')
+export class Lesson {
+    @PrimaryGeneratedColumn()
+      id!: bigint;
+
+    @ManyToOne(() => Section)
+      section!: Section;
+
+    @Column()
+      name!: string;
+
+    @Column('text')
+      description!: string;
+
+    @Column('integer')
+      time!: number;
+
+    @Column({ type: 'datetime' })
+      created_at!: Date;
+
+    @Column({ type: 'datetime' })
+      updated_at!: Date;
+
+    constructor(lessonData?: Partial<Lesson>) {
+      this.name = lessonData?.name ?? '';
+      this.description = lessonData?.description ?? '';
+      this.time = lessonData?.time ?? 0;
+      this.created_at = lessonData?.created_at ?? new Date();
+      this.updated_at = lessonData?.updated_at ?? new Date();
+    }
```

Comment:
```suggestion
    constructor(lessonData?: Partial<Lesson>) {
      lessonData && Object.assign(this, lessonData);
    }
```
Những entities này e fix tương tự